### PR TITLE
Mixpanel location

### PR DESCRIPTION
**Pull request by volovyk-s**
image

### CODE REVIEW

File: utils/eventtracking.js [hunk 1]
```diff
@@ -82,6 +86,16 @@ const track = async (eventType, eventProperties, options) => {
...
+    ).then(
+        res => res.text()
+    ).then(
+        ip => ip.trim()
+    );
+}
+
 const getEventTrackingConsent = async () => {
     return askYesNoQuestion(
         chalk`Please help us to collect data on near-cli usage to improve developer experience. ` +
```
Comment (TiffanyGYJ) [inline utils/eventtracking.js:101]: Is this the only question for getting consent? Do we have separate one when we grab their account id?

**Review by TiffanyGYJ — APPROVED** [review]

File: utils/eventtracking.js (hunk 1 above)
Comment (volovyk-s) [reply utils/eventtracking.js:101]: This is the only question that is getting asked on the first run.
We are not collecting NEAR Account ID, we are collecting generated user ID. It's unique for each machine and can be easily deleted by the user.

**Comment by TiffanyGYJ** [comment]
Any reason why the checks are failed? Is that why the change has not been merged?

**Comment by volovyk-s** [comment]
@TiffanyGYJ Travis is dead for some reason. Trying to figure out the reason. I will release it right after that.

**Pull request merged**